packages/lesswrong/components/posts/PostsNewForm.tsx

```diff
@@ -115,7 +115,9 @@ const PostsNewForm = ({classes}: {
   const af = forumTypeSetting.get() === 'AlignmentForum'
   const prefilledProps = {
     isEvent: query && !!query.eventForm,
+    activateRSVPs: true,
```

Comment:
Say more about this choice?

Reply:
Ah I should have included a link to [this](https://cea-core.slack.com/archives/CJZ7QMY5V/p1635999702034700) slack thread, where Ruby shares some user feedback about the Community page. That's also where I got the idea to add the note when creating a new comment.

I figured this is also useful for us, because we probably want the contractor to enable RSVPs for all the events they add.